src/sst.rs

```diff
@@ -253,6 +253,19 @@ impl SsTableFormat {
             self.compression_codec,
         )
     }
+
+    pub(crate) fn estimate_sst_size(
```

Comment:
I had debated whether this should be static (SsTableFormat::estimate_sst_size). It would make this PR less invasive to the DbInner struct. But I think I like having the SsTableFormat in DbInner anyway, so I'm good keeping it as-is.